torch_geometric/datasets/linkx_dataset.py

```diff
@@ -32,23 +32,45 @@ class LINKXDataset(InMemoryDataset):
             being saved to disk. (default: :obj:`None`)
     """
 
-    url = 'https://github.com/CUAI/Non-Homophily-Large-Scale/raw/master/data'
+    github_url = 'https://github.com/CUAI/Non-Homophily-Large-Scale/' \
+                 'raw/master/data'
+    gdrive_url = 'https://drive.google.com/uc?confirm=t&'
```

Comment:
just a note on the `confirm=t` - this skips the virus check  which cannot be performed on larger files